lectures/geom_series.md

```diff
@@ -11,24 +11,9 @@ kernelspec:
   name: python3
 ---
 
-(geom_series)=
-```{raw} html
```

Comment:
@HengchengZhang removing this will remove it from the download notebook. I wonder if we should use `{raw} jupyter` to include it int he download notebook via `sphinx-tojupyter`

Reply:
Hi @mmcky I think it's still working fine in the downloaded pdf of the notebook. (In some lectures we are still using this settings but most are not.) It's another inconsistent setting found.

Reply:
I am referring to the download `ipynb` that are synced to https://github.com/quantecon/lecture-python-intro.notebooks